manifests/monolithic.pp

```diff
@@ -0,0 +1,18 @@
+# **NOTE: THIS IS A [PRIVATE](https://github.com/puppetlabs/puppetlabs-stdlib#assert_private) CLASS**
+#
+# This is simply present to isolate the logic of the installation from the
```

Comment:
Seems like the purpose of this class is to prevent connection configuration/service resources from being instantiated when only instance configuration/service resources are required.  If so, I'm not sure I follow the comment.

Reply:
I can use your wording if that will help people understand what's going on.